src/Chapter2.hs

```diff
@@ -336,7 +336,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList from to list
+  | from < 0 || to < 0 || to < from = []
+  | otherwise = drop from(take (to + 1) list)
```

Comment:
Nice and solid implementation! The only stylistic suggestion I would give is to add space before the second argument of `drop`. Usually arguments are space-separated in Haskell, even if they can be grouped with `()`
```suggestion
  | otherwise = drop from (take (to + 1) list)
```

Reply:
:handshake: